examples/tutorials/lighting/app.ts

```diff
@@ -146,6 +157,7 @@ export default class AppAnimationLoopTemplate extends AnimationLoopTemplate {
   shaderInputs = new ShaderInputs<{
     app: typeof app.props;
     lighting: typeof lighting.props;
+    // Can replace with gouraudMaterial
```

Comment:
Was the idea that we would only have one module (and this one material) for phong and gouraud?

Reply:
I didn't meant to remove `gouraudMaterial`, just to share the code between the two materials better

Reply:
Agreed, that is a good first step. Suspecting that we can unify into one module. Maybe just control with a uniform or expose both set of vertex and fragment functions from one module. For later.